### DISCUSSION TITLE
Can not able to apply padding to our own html element

### DISCUSSION THREAD

**Discussion by Prshant-Sharma** [hakimio/ngx-ui-tour#144] [Q&A] [answered]
I'm using `ngx-ui-tour-md-menu` of version 8.1.2 

How can I apply custom CSS to the HTML element on which `tourAnchor` is applied?

Basically, I want to give some padding and want to round the corners of the element where UI tour is shown for better looks. I can't use `backdropConfig` option as I'm using version 8 of ngx-ui-tour-md-menu and this option is not available for this version.

I followed [documentation](https://hakimio.github.io/ngx-ui-tour/md-menu/FAQ#backdrop-padding) as well but it doesn't work for me as shown below:

<img width="185" alt="image" src="https://github.com/hakimio/ngx-ui-tour/assets/63813857/91f4fe7f-e1e5-4152-8fb1-5fd6f7b35301">

I'm using this below snippet of CSS in my code to give padding to the button `Let's Start`. Just like its shown in left-right, same I want from top and bottom too.

<img width="331" alt="image" src="https://github.com/hakimio/ngx-ui-tour/assets/63813857/009b476e-6908-4c27-b9b1-601d8c903700">

Hope you understand the challenge that  I'm facing to give custom CSS.
Looking forward for your assistance here.

**Comment by hakimio** [comment, marked as answer]
In case of a button, you might have to nest it in another `div` element and set `tourAnchor` to the new `div` container to get the desired result.
Check out [this demo project](https://stackblitz.com/edit/angular-wau2z8-ggpcaf?file=src%2Fapp%2Fui-tour-demo.component.html,src%2Fapp%2Fui-tour-demo.component.scss) for an example of how it can be done.

**Reply by Prshant-Sharma** [reply]
@hakimio, thanks for your quick response!

I tried the same way that you suggested but the problem I got to know is if I change the window size then its showing perfectly, means if I open the console window below then it take all the padding correctly but initially, its not showing as it should be!

Below is the initial screenshot:

<img width="804" alt="image" src="https://github.com/hakimio/ngx-ui-tour/assets/63813857/1aa4a21f-26de-49dc-96fc-96bf37f5d62b">

After opening the just console window below, its showing fine:

<img width="800" alt="image" src="https://github.com/hakimio/ngx-ui-tour/assets/63813857/201438e4-2876-41cb-87e4-8d39c37bfbfd">

**Comment by hakimio** [comment]
Maybe your button is not fully rendered when the tour is trying to get its bounding rectangle. You can try to set `delayBeforeStepShow` option to see if it helps.
If you can reproduce the issue with the latest version of the library, feel free to submit a new bug report with reproduction of the issue.
